e2eTests/package-lock.json

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
Is this file accidentally checked in? We have not merged the playwright pr right? Should probably remove this file and use the package-lock.json from the other pr

Reply:
Also once you merge the playwright pr. 

Would be good to add in browser end to end tests as well for this